LayoutTests/fast/images/imageDocument-createDocumentStructure-crash.html

```diff
@@ -0,0 +1,32 @@
+<script>
+    if (window.testRunner)
```

Comment:
Looks like this can be reduced further.

Reply:
per offline sync this test case is not reliably reproduce the issue, updated patch to remove test case for now.